python/dask_cudf/dask_cudf/tests/test_groupby.py

```diff
@@ -274,7 +274,13 @@ def test_groupby_multiindex_reset_index():
     )
     ddf = dask_cudf.from_cudf(df, npartitions=1)
     pddf = dd.from_pandas(df.to_pandas(), npartitions=1)
-    gr = ddf.groupby(["a", "c"]).agg({"b": ["count"]}).reset_index()
+    gr = (
+        ddf.groupby(["a", "c"])
+        .agg({"b": ["count"]})
+        .reset_index()
+        .sort_values(by=["a", "c"])
+        .reset_index(drop=True)
+    )
     pr = pddf.groupby(["a", "c"]).agg({"b": ["count"]}).reset_index()
```

Comment:
Should do the same sorting for Pandas here as well.

Reply:
I tried to add same to pandas, but it was giving errors saying methods are not there. So, moved it after compute for both to be look same.